ffi/capi/src/properties_enums.rs

```diff
@@ -1568,6 +1568,51 @@ pub mod ffi {
         }
     }
 
+    #[diplomat::rust_link(icu::properties::props::IndicConjunctBreak, Struct)]
+    #[diplomat::enum_convert(icu_properties::props::IndicConjunctBreak, needs_wildcard)]
+    pub enum IndicConjunctBreak {
```

Comment:
issue: if we're considering the API unstable doc-hidden, it should not be stable on FFI. we currently don't have a way to do doc(hidden) in FFI, so remove?

Reply:
I will remove FFI. This is hidden so I guess it is unnecessary to update `ffi/capi/tests/missing_apis.txt`.